GoogleForCreators/web-stories-wp
# Fixed lasso selection

## Context

Fixes lasso selection by taking both the offsets of the full-bleed container and the page inside it into account.

Also re-enables tests, that know work as expected.

## User-facing changes

Lasso selection now works - even when zoomed!

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Add an element
2. Lasso-select it in different ways and observe that the element is correctly selected or not whether it's inside the lasso or not.
3. Change zoom to 200%
4. Scroll down and right a bit
5. Repeat lasso selection and confirm that edges are found correctly.
6. Repeat for multiple elements and multiple zoom views

### UAT




## Checklist




---

Fixes #6911
